bitrise.yml

```diff
@@ -31,7 +31,7 @@ workflows:
     steps:
     - activate-ssh-key@4: {}
     - git-clone@8: {}
-    - path::./:
+    - git::https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git:
```

Comment:
we want to keep `path::./`, otherwise we'll be testing the `main` branch even on PRs